modules/tinymce/src/core/test/ts/browser/content/EditorContentTest.ts

```diff
@@ -92,15 +89,17 @@ describe('browser.tinymce.core.content.EditorContentTest', () => {
   // TODO: TINY-8367 The table plugin code has been moved to core so there is now always an extra DIV in the dom for the table resize bars
```

Comment:
Is this comment still accurate? As we've changed how things are retrieved I don't believe it's valid anymore. TINY-8367 could also be closed/linked to this fix if it's not longer valid as well.

Reply:
No, I think this is now properly irrellevant.